tests/integration/models/test_model_forward.py

```
@@ -338,6 +338,16 @@ def test_varying_down_sample_ratio(self, down_sample_ratio):
         assert loss.isfinite()
         loss.backward()
 
+    @pytest.mark.parametrize("max_anchors", [4, 16, None])
+    def test_varying_max_anchors(self, max_anchors):
+        model = make_peagle_model(max_anchors=max_anchors)
+        samples = _make_samples([128])
+        batch = make_batch(max_len=MAX_LEN, samples=samples, hidden_size=HIDDEN_SIZE)
+        draft_tokens, loss, metrics = model(**batch)
```

Ruff (0.15.17)

[warning] Unpacked variable `draft_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)